installation_and_upgrade/Jenkinsfile_vhd_creation

```diff
@@ -0,0 +1,48 @@
+#!groovy
+
+pipeline {
+
+  // agent defines where the pipeline will run.
+  agent {  
+    label {
+      label "create_vhd"
+    }
+  }
+  
+  triggers {
+    pollSCM('H/2 * * * *')
+    upstream(upstreamProjects: 'ibex_gui_pipeline,genie_python_pipeline,EPICS_IOC_Windows7_x64,EPICS_IOC_Windows7_x64_CLEAN', threshold: hudson.model.Result.SUCCESS)
+  }
+  
+  stages {  
+    stage("Checkout") {
+      steps {
+        echo "Branch: ${env.BRANCH_NAME}"
+        checkout scm
+      }
+    }
+    
+    stage("Build") {
+      steps {
+        bat """
+            set BUILD_NUMBER=${env.BUILD_NUMBER}
+            installation_and_upgrade/vhd_build.bat
+            """
+      }
+    }
+  }
+  
+  post {
+    failure {
+      step([$class: 'Mailer', notifyEveryUnstableBuild: true, recipients: 'icp-buildserver@lists.isis.rl.ac.uk', sendToIndividuals: true])
+    }
+  }
+  
```

Comment:
We now use office365ConnectorWebhooks in options section rather than email - see JenkinsFile in EPICS for syntax and notification control